rca/project_styleguide/templates/patterns/organisms/programme_tabs/requirements.html

```diff
@@ -1,5 +1,8 @@
 <section class="section section--start bg bg--dark">
     Requirements content here
+{{ page.requirements_subtitle }}
+{{ page.requirements_text }}
+{{ page.requirements_blocks }}
```

Comment:
Even if it's sample, can you please use `{% include_block %}` in case front-end gets to this?